spacecmd/src/spacecmd/utils.py

```diff
@@ -370,11 +376,21 @@ def parse_time_input(userinput=''):
     logging.error(_N('Invalid time provided'))
     return None
 
-
-# Compares 2 package objects (dicts) and returns the newest one.
-# If the objects are the same, we return None
 def latest_pkg(pkg1, pkg2, version_key='version',
                release_key='release', epoch_key='epoch'):
+    """
+    Compares 2 package objects (dicts) and returns the newest one.
+
+        Parameters:
+            pkg1 (dict): A dictionary
```

Comment:
The `pkg1` and `pkg2` parameters are objects different from dictionaries. I think [this](https://github.com/uyuni-project/uyuni/blob/0366d51b3ab3286fd73c18f96d7080e39cefd8b7/java/code/src/com/redhat/rhn/frontend/xmlrpc/system/SystemHandler.java#L1284) is an example. However, you could just write that pkg1 is an object describing a package.